### PR TITLE
Decouple tinyxml

### PR DESCRIPTION
**Pull request by alexgdi**
tinyxml was decoupled.  Upgraded to 2.6.2. The compilation is successful, need a review.

### CODE REVIEW

**Comment by Xottab-DUTY** [comment]
It seems that you have pushed not all files..

**Comment by alexgdi** [comment]
@Xottab-DUTY its all, I did not include lib, i mush add lib to submodule or push them all?

**Comment by Xottab-DUTY** [comment]
You can include submodule into this pull request.

The main thing: if its working, then fine.
But it would be nice if project references would be used instead of directly including tinyxml.lib into AdditionalDependencies

Also, what with OpenAL?

**Comment by alexgdi** [comment]
@Xottab-DUTY, no problem with OpenAL. All problems solved. TinyXML not have a git repo. maybe we must host repo on OpenXRay organization? TinyXML have zlib\libpng licence.

**Comment by Xottab-DUTY** [comment]
Ah, i think you also updated it to the new [TinyXML2](https://github.com/leethomason/tinyxml2). Well, we can do this later.
You can host repo in your own account. We already using your lzo repo, so, I think there's no problem.

**Comment by alexgdi** [comment]
done.

**Comment by Xottab-DUTY** [comment]
One last thing needed: adding tinyxml project file to the engine.sln.

After this we both have to wait until PR will be merged =)

**Comment by alexgdi** [comment]
not build, got an error - OutPath not found.

File: src/engine.sln
Comment (andrew-boyarshin) [inline src/engine.sln:2]: Whoa, that's a change. Probably better to wait with that change.

**Review by andrew-boyarshin — CHANGES_REQUESTED** [review]
Visual Studio version upgrade should not be included in this commit.

**Comment by alexgdi** [comment]
fixed.

File: src/engine.sln
Comment (Xottab-DUTY) [reply src/engine.sln:2]: Will you revert _this_ change to the solution file? We can't change the VS version, since current branch is not ready for it.

File: src/xrCore/xrCore.vcxproj
Comment (Xottab-DUTY) [inline src/xrCore/xrCore.vcxproj:21]: Windows target version is unset for all the project files. This is not critical, but it's recommended to not change this.

File: src/xrGame/xrGame.vcxproj
Comment (Xottab-DUTY) [inline src/xrGame/xrGame.vcxproj:20]: Windows target version is unset for all the project files. This is not critical, but it's recommended to not change this.

File: src/xrCore/xrCore.vcxproj
Comment (Xottab-DUTY) [inline src/xrCore/xrCore.vcxproj:95]: This is not critical, but it's recommended to use project references instead of explicit pointing the lib name to the linker.
Also you have dublicated the lib name here.

**Review by Xottab-DUTY — CHANGES_REQUESTED** [review]
It is stable enough? Have you tested it for a long time?

**Pull request closed without merging**
